rifle.py

```diff
@@ -36,17 +40,49 @@ def __init__(self, universe, DRAWING_SCALE):
         self.word_length_min_ = 3
         self.word_length_range = 3
 
-
+    def getTargetID(self,input): 
+        for enemy in self.universe_.enemies(): 
+            target_tag_word = self.target_tags_[enemy.ID_]
```

Comment:
`target_tag_word` is only used in the next if statement, so should be fine to just put these 2 lines together. 
As in:
`if input == self.target_tags_[enemy.ID_]:`